React/CoreModules/RCTDevSettings.mm

```diff
@@ -196,6 +202,29 @@ - (void)initialize
                          queue:dispatch_get_main_queue()
                      forMethod:@"reload"];
   }
+    
+#if RCT_DEV_MENU
+  if (self.bridge) {
+    RCTBridge *__weak weakBridge = self.bridge;
+    _bridgeExecutorOverrideTokenDevMenu = [[RCTPackagerConnection sharedPackagerConnection]
+        addNotificationHandler:^(id params) {
+          if (params != (id)kCFNull && [params[@"debug"] boolValue]) {
+              weakBridge.executorClass = self.executorClass;
+          }
+        }
+                         queue:dispatch_get_main_queue()
+                     forMethod:@"devMenu"];
+  }
+
+  if (numInitializedDevMenuModules++ == 0) {
+    reloadTokenDevMenu = [[RCTPackagerConnection sharedPackagerConnection]
```

Comment:
I think you actually want to group this with the `reloadToken` setup above.

The point of this variable is to initialize the handler the very first time init runs here.

Reply:
We probably want something like this:

```objc
  if (numInitializedModules++ == 0) {
    reloadToken = [[RCTPackagerConnection sharedPackagerConnection]
        addNotificationHandler:^(id params) {
          RCTTriggerReloadCommandListeners(@"Global hotkey");
        }
                         queue:dispatch_get_main_queue()
                     forMethod:@"reload"];
#if RCT_DEV_MENU
    devMenuToken = [[RCTPackagerConnection sharedPackagerConnection]
        addNotificationHandler:^(id params) {
            [self.bridge.devMenu show];
        }
                        queue:dispatch_get_main_queue()
                    forMethod:@"devMenu"];
#endif
  }